widgets/hero/hero.php

```diff
@@ -261,7 +266,7 @@ function get_widget_form(){
 	function get_frame_background( $i, $frame ){
 		$background_image = siteorigin_widgets_get_attachment_image_src(
 			$frame['background']['image'],
-			'full',
+			$frame['background']['size'],
```

Comment:
This needs an `empty` check and should fall back to using `'full'` for existing widget instances which won't yet have the `$frame['background']['size']` property.